src/Examine.Lucene/ReaderStatus.cs

```diff
@@ -1,9 +1,27 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 
 namespace Examine.Lucene
 {
-    public enum ReaderStatus { Current, Closed, NotCurrent }
+    /// <summary>
+    /// TODO: Is this enum used? (Found 0 references)
```

Comment:
Looks like this is old, can't remember what from

Reply:
@Shazwazza Should we delete it then or is it best to keep?

Reply:
We should prob delete it